pydantic/fields.py

```diff
@@ -33,7 +34,6 @@
 from .types import Json, JsonWrapper
 from .typing import (
     Callable,
```

Comment:
I guess we could do the same for `Callable` and here import it from `collections.abc` but I'm not sure.

Reply:
Also worth considering them to be removed from `__all__` in `pydantic._typing`.